django/db/models/expressions.py

```diff
@@ -866,6 +895,112 @@ def as_sql(self, compiler, connection, template=None, case_joiner=None, **extra_
         return sql, sql_params
 
 
+class SubQuery(Expression):
+    """
+    An explicit subquery. It may contain OuterRef() references to the outer
+    query, which will be resolved when it is applied to that query.
+    """
+    template = '(%(subquery)s)'
+
+    def __init__(self, subquery, output_field=None, **extra):
+        self.subquery = subquery.all()
```

Comment:
I guess the other place it could be changed is in `relabelled_clone`, but the `.subquery.query` is cloned by the `super()` call. I'm happy to remove it.